en/docs/insights/graphiql-explorer.md

```diff
@@ -2,13 +2,13 @@
 template: templates/graphiql-explorer.html
 ---
 
-# GraphiQL Explorer
+# GraphQL Explorer
 
 Start exploring Choreo Insights API queries using your data now.
```

Comment:
Suggestion:
"You can use the GraphQL explorer to test Choreo Insights API queries using your data.  Try out various queries to inspect the responses."

Reply:
"You can use the GraphQL Explorer to test Choreo Insights API **operations** using your data. Try out various queries to inspect the responses." will give the correct meaning.